app/src/main/kotlin/com/flxrs/dankchat/data/NotificationService.kt

```diff
@@ -26,7 +26,10 @@ import com.flxrs.dankchat.data.twitch.message.WhisperMessage
 import com.flxrs.dankchat.main.MainActivity
 import dagger.hilt.android.AndroidEntryPoint
 import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.forEach
```

Comment:
Unused import I think?

Reply:
Fixed in [16801e8](https://github.com/flex3r/DankChat/pull/309/commits/16801e8d12a3c54426e842693c8b70b082eeebd5)